src/app/@ansyn/menu-items/tools/components/image-processing-control/image-processing-control.component.ts

```diff
@@ -93,4 +98,27 @@ export class ImageProcessingControlComponent {
 	close() {
 		this.expand = false;
 	}
+
+	ngOnInit(): void {
+		this.resetParams();
+		this.store$.select(toolsStateSelector)
+			.withLatestFrom(this.store$.select(mapStateSelector), this.store$.select(toolsStateSelector))
+			.subscribe(res =>
+			{
+				if (res[0].imageProcessingHash[res[1].activeMapId] !== undefined) {
+					this.params.forEach(param => {
```

Comment:
instead of `res[0]` and `res[1]` you can init variables..

const [value1, value2] = res